node_agent/main.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Istio Authors
```

Comment:
"You can avoid having the na package insider node_agent dir, which is non-ideal."

Not sure why do we want to avoid the "na" package. If it is just a naming issue, I can change it to "app".